include/fmt/chrono.h

```diff
@@ -458,6 +456,7 @@ template <typename Char> struct formatter<std::tm, Char> {
   basic_string_view<Char> specs;
 };
 
+FMT_MODULE_EXPORT_END
 namespace detail {
```

Comment:
I suggest adding `FMT_BEGIN_DETAIL_NAMESPACE` / `FMT_END_DETAIL_NAMESPACE` macros to avoid repeating this sequence throughout the code.

Reply:
Makes sense. I will apply this suggestion in all of `fmt/include` where applicable.